tools/cli/commands.js

```diff
@@ -1413,6 +1413,10 @@ testCommandOptions = {
     deploy: { type: String },
     production: { type: Boolean },
     settings: { type: String },
+    // Indicates whether these self-tests are running headless, e.g. in a
+    // continuous integration testing environment, where visual niceties
+    // like progress bars and spinners are unimportant.
```

Comment:
I debated on whether or not to even keep this documentation, and of course I left a minor copy/paste error.  Let me know if you would rather just strip this comment altogether.